### PR TITLE
Add some more HockeyApp logging

### PR DESCRIPTION
**Pull request by alinapopa**

### CODE REVIEW

File: src/MobileApps/MyDriving.sln [hunk 1]
```diff
@@ -629,7 +628,6 @@ Global
 		{B09661C2-82A6-4D16-B4FD-7B23D0FFE1FE}.Debug|iPhone.Deploy.0 = Debug|Any CPU
 		{B09661C2-82A6-4D16-B4FD-7B23D0FFE1FE}.Debug|iPhoneSimulator.ActiveCfg = Debug|Any CPU
 		{B09661C2-82A6-4D16-B4FD-7B23D0FFE1FE}.Debug|iPhoneSimulator.Build.0 = Debug|Any CPU
-		{B09661C2-82A6-4D16-B4FD-7B23D0FFE1FE}.Debug|iPhoneSimulator.Deploy.0 = Debug|Any CPU
```
Comment (xiangyan99) [inline src/MobileApps/MyDriving.sln]: why delete it?

File: src/MobileApps/MyDriving/MyDriving.AzureClient/DeviceProvisionHandler.cs [hunk 2]
```diff
@@ -28,7 +28,7 @@ private DeviceProvisionHandler()
             catch (Exception e)
             {
                 //Occassionally, a System.IO.FileLoadException can occur
-                Logger.Instance.WriteLine("Unable to get device id and/or host name from user settings: " + e.Message);
+                Logger.Instance.Track("Unable to get device id and/or host name from user settings: " + e.Message);
```
Comment (xiangyan99) [inline src/MobileApps/MyDriving/MyDriving.AzureClient/DeviceProvisionHandler.cs]: What's the difference?

File: src/MobileApps/MyDriving/MyDriving.Shared/PlatformLogger.cs [hunk 3]
```diff
@@ -22,12 +22,6 @@ public override void Identify(string uid, IDictionary<string, string> table = nu
             base.Identify(uid, table);
         }
 
-        public override void WriteLine(string line)
```
Comment (xiangyan99) [inline src/MobileApps/MyDriving/MyDriving.Shared/PlatformLogger.cs]: why delete it?
We can keep both. One for server logging, one for local debugging, right?

File: src/MobileApps/MyDriving/MyDriving/ViewModel/CurrentTripViewModel.cs [hunk 4]
```diff
@@ -194,9 +194,14 @@ public string EngineLoad
             if (IsRecording)
                 return false;
 
-            var track = Logger.Instance.TrackTime("SaveRecording");
```
Comment (xiangyan99) [inline src/MobileApps/MyDriving/MyDriving/ViewModel/CurrentTripViewModel.cs]: We use this to measure performance. Why delete it?

File: src/MobileApps/MyDriving.sln (hunk 1 above)
Comment (alinapopa) [inline src/MobileApps/MyDriving.sln]: That was not intentional. I'll add it back.

File: src/MobileApps/MyDriving/MyDriving.Shared/PlatformLogger.cs (hunk 3 above)
Comment (alinapopa) [inline src/MobileApps/MyDriving/MyDriving.Shared/PlatformLogger.cs]: It is not needed. PlatformLogger.Track() also does Debug.WriteLine().

File: src/MobileApps/MyDriving/MyDriving/ViewModel/CurrentTripViewModel.cs (hunk 4 above)
Comment (alinapopa) [inline src/MobileApps/MyDriving/MyDriving/ViewModel/CurrentTripViewModel.cs]: After switching from Xamarin Insights to HockeyApp, TrackTime() does not do anything. HockeyApp does not have a similar api.

Comment (xiangyan99) [inline src/MobileApps/MyDriving/MyDriving/ViewModel/CurrentTripViewModel.cs]: Yes. But it does not do harm to keep them and enable them when HockeyApp supports it?

Comment (alinapopa) [inline src/MobileApps/MyDriving/MyDriving/ViewModel/CurrentTripViewModel.cs]: Added back the time tracking.

**Pull request merged**